js/main.js

```diff
@@ -1,2 +1,29 @@
 /* global data */
 /* exported data */
+
+function handlePhoto(event) {
+  var userUrl = event.target.value;
+  $img.setAttribute('src', userUrl);
+}
+
+var $photoUrl = document.querySelector('#photo-url');
+var $img = document.querySelector('img');
+$photoUrl.addEventListener('input', handlePhoto);
+
+function handleSubmit(event) {
+  event.preventDefault();
+  var entry = {};
+  entry.title = $form.elements[0].value;
+  entry.photoUrl = $form.elements[1].value;
+  entry.note = $form.elements[2].value;
```

Comment:
Please use the named properties rather than the numbered indexes, it is easier to read and less error-prone